email/sender.go

```diff
@@ -0,0 +1,267 @@
+package email
+
+import (
+	"context"
+	"crypto/tls"
+	"net/url"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+
+	"github.com/sirupsen/logrus"
+	lock "github.com/square/mongo-lock"
+	"gitlab.com/NebulousLabs/errors"
+	"gitlab.com/SkynetLabs/skyd/skymodules"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"gopkg.in/mail.v2"
+)
+
+const (
+	// batchSize defines the largest batch of emails we will try to send.
+	batchSize = 10
+
+	// lockTTL defines the TTL (in seconds) of the lock on the emails waiting to
+	// be sent.
+	lockTTL = 30
+
+	// sleepBetweenScans defines how long the sender should sleep between its
+	// sweeps of the DB.
+	sleepBetweenScans = 10 * time.Second
```

Comment:
seems high

Reply:
What would be a more adequate value? 1 second?